Parse-Dashboard/index.js

```diff
@@ -38,34 +38,44 @@ packageJson('parse-dashboard').then(latestPackage => {
 const port = program.port || process.env.PORT || 4040;
 const allowInsecureHTTP = program.allowInsecureHTTP || process.env.PARSE_DASHBOARD_ALLOW_INSECURE_HTTP;
 
+let explicitConfigFileProvided = !!program.config;
 let configFile = null;
 let configFromCLI = null;
 if (!program.config) {
-  if (!program.appId || !program.masterKey || !program.serverURL) {
-    console.log('You must provide either a config file or an app ID, Master Key, and server URL. See parse-dashboard --help for details.');
-    process.exit(4);
-  }
-  configFromCLI = {
-    data: {
-      apps: [
-        {
-          appId: program.appId,
-          serverURL: program.serverURL,
-          masterKey: program.masterKey,
-          appName: program.appName,
-        }
-      ]
-    }
+  if (program.serverURL && program.masterKey && program.appId) {
+    configFromCLI = {
+      data: {
+        apps: [
+          {
+            appId: program.appId,
+            serverURL: program.serverURL,
+            masterKey: program.masterKey,
+            appName: program.appName,
+          },
+        ]
+      }
+    };
+  } else if (!program.serverURL && !program.masterKey && !program.appName) {
+    configFile = path.join(__dirname, 'parse-dashboard-config.json');
   }
 } else {
-  configFile = program.config || path.join(__dirname, 'parse-dashboard-config.json');
+  configFile = program.config;
   if (program.appId || program.serverURL || program.masterKey || program.appName) {
     console.log('You must provide either a config file or app ID, Master Key, and server URL, not both.');
```

Comment:
You must provide either a config file or required CLI options (app ID, Master Key, server URL); not both.

Think this would be more clear.